### ISSUE TITLE
vmagent: json stats for remoteWrite.url's 

### ISSUE THREAD

**Issue by psvmcc** [VictoriaMetrics/VictoriaMetrics#4890]
### Describe the solution you'd like

Additional stats for remoteWrite.url's monitoring, like /api/v1/remotes with json details about each remote storage sync details, and global status if one of them is not synced for some reason. 

```
{
    "status": "failure",
    "remotes" : [
        {
            "Name":"1:secret-url",
            "ReaderOffset":0,
            "WriterOffset":0,
            "status", "success",
            "errors_count": 0
        },
        {
            "Name":"2:secret-url",
            "ReaderOffset":0,
            "WriterOffset":76171,
            "status", "failure",
            "errors_count": 253
        },
    ]
}
```

PS: I know we can collect metrics from vmagent and alert on it, but sometimes its not possible :(

### Additional information

💙💛

**Comment by AVESH-RAI** [comment]
I am interested in contributing for this requirement, if this feature is good to go.

**Comment by AVESH-RAI** [comment]
@hagen1778 I am willing to contribute here, can you assign this to me.

**Comment by hagen1778** [comment]
> PS: I know we can collect metrics from vmagent and alert on it, but sometimes its not possible :(

Could you please elaborate more on this? If you can call "/api/v1/remotes" then it is likely you can call to "/metrics" as well.

**Comment by psvmcc** [comment]
Yes, I can query `/metrics`, but its hard to parse it, and create alert with other monitoring system(like nagios/icinga). With json I can just check `status` element in dict, and alert on failure. Thanks!

**Comment by hagen1778** [comment]
Thanks for the explanation!

Looks like this problem can be solved with a custom reverse proxy, which would read&parse `/metrics` response to provide a json output for your services.

The new features can be added if they provide unique functionality or significantly simplify existing ones. So far, I'm not convinced that this is the case.
